GOVERNANCE.md

```diff
@@ -151,7 +151,8 @@ The nomination passes if no Collaborators oppose it after one week. Otherwise,
 the nomination fails.
 
 There are steps a nominator can take in advance to make a nomination as
-frictionless as possible. Use the [Collaborators discussion page][] to request
+frictionless as possible. Use the [Collaborators discussion page][](only
+Collaborators have permissions to view) to request
```

Comment:
I think the parenthetical makes the sentence harder to read. Maybe this instead?:

```suggestion
frictionless as possible. Use the private [Collaborators discussion page][] to
request
```

Is "private" sufficiently clear in this case?

Reply:
Sorry. I don't think so. It will make me feel confused when I read `private`. I think my remark is more clear than `private`.

Reply:
> I think my remark is more clear than `private`.

That's true. What you wrote is more clear than my suggestion.

What I was trying to avoid was separating "Use the Collaborators discussion board" from what to use it for: "to request feedback from other Collaborators in private." 

Current sentence: "Use the Collaborators discussion board to request feedback from other Collaborators in private." 

With change here: "Use the Collaborators discussion board (only Collaborators have permissions to view) to request feedback from other Collaborators in private."

I feel like that makes the resulting sentence harder to understand.

How about this?: 

> Use the Collaborators discussion board to requeist feedback from other Collaborators in private. Only Collaborators can view the private discussion board.

Or maybe this?:

> To request feedback from other Collaborators in private, use the Collaborators discussion board (which only Collaborators may view).

I'm open to other ideas if these suggestions are no good. I'm also OK if this lands as-is, but I do *prefer* to not have a parenthetical clause as an aside in the middle of the sentence.

Reply:
Thank you for your advice. I think the second idea is pretty good.